draft-barnes-hpke-hpke.md

```diff
@@ -1156,7 +1156,7 @@ unlikely to be reached in practical applications. Future specifications
 that define new KDFs MUST specify bounds for these variable-length
 parameters.
 
-The RECOMMENDED limit for these values is 64 bytes. This would enable
+The RECOMMENDED limit for these values is 66 bytes. This would enable
```

Comment:
This one took me an age to understand.  The section is talking about input length, with a table of input lengths that contains very large numbers, but what this refers to is the length of the prefix label.

```suggestion
The RECOMMENDED limit for these prefixes is 66 bytes. This would enable
```

Do the numbers in the table need to be tweaked to match?

Reply:
No, I think this is about the values, not the prefixes.  In the sense that, while the bounds above are huge, in practice, you basically never need something more than 64 bytes.  (64 bytes should be enough for anyone!)  I'll try rewording to clarify this.